### PR TITLE
No retries in build and run lambda

### PR DESCRIPTION
**Pull request by bencodeorg**
Lambda by default will retry twice on timeout or error -- this sets it such that we do not retry. This will be particularly useful if we start using `System.exit` (which will error) after all projects.

Deployment might be a little tricky, as on my dev deploy this setting did not stick until I fully deleted/recreated my dev deploy.

A proposal would be to deploy this change, then change the setting manually. If we do this, we should check the next time that we deploy to make sure that the setting sticks.

### CODE REVIEW

File: template.yml [hunk 1]
```diff
@@ -284,7 +284,9 @@ Resources:
         ProvisionedConcurrentExecutions: !Ref ProvisionedConcurrentExecutions
       Description: Compile and execute a JavaLab project.
       MemorySize: 1769
-      Timeout: 300
+      Timeout: 30
```
Comment (molly-moen) [inline template.yml:287]: this should still be 300 right?

Comment (bencodeorg) [reply template.yml]: yes 😅 , thanks

**Review by molly-moen — APPROVED** [review]
LGTM!

**Review by jmkulwik — APPROVED** [review]
LGTM!

**Pull request merged**